src/assets/data/resources.json

```diff
@@ -175,5 +175,33 @@
       "url": "https://skills.github.com/",
       "tags": ["git", "github"],
       "image": "https://user-images.githubusercontent.com/1221423/156894097-ff2d6566-7b6a-4488-950e-f4ebe990965a.svg"
+    },
+    {
+        "title": "Pro Git",
+        "image": "https://git-scm.com/images/progit2.png",
+        "description": "The guide to Git and its usage.",
+        "url": "https://git-scm.com/book/en/v2",
+        "tags": ["git", "books", "documentation", "developer"]
+    },
+    {
+        "title": "Eloquent JavaScript",
+        "image": "https://nostarch.com/sites/default/files/styles/uc_product_full/public/EJS3_front.png?itok=2qHtZpJa",
```

Comment:
Kindly change the image link with a proper github hosted .png image link. Thanks.

Reply:
![EJS3_front](https://user-images.githubusercontent.com/68626833/198849739-bd1b61bc-c813-432d-9622-b8b759c31185.png)
![1_SwbEdd_9O-VkpkqeTS72VA](https://user-images.githubusercontent.com/68626833/198849798-9e1b8173-e36d-4e66-b49c-ef2b47363b24.png)
![how-to-contribute-to-an-open-source-project-on-github](https://user-images.githubusercontent.com/68626833/198849864-46c5cdf5-8f08-47e7-858e-19abfffb2599.png)